students/tonylee/FizzBuzz.py

```diff
@@ -0,0 +1,13 @@
+def printFizzBuzz():
+	for number in range(1, 100):
+		if number%3 == 0 and number%5 == 0:
+			print "FizzBuzz"
```

Comment:
Looks like this was written for Python 2.7 based on this print statement.  Please use Python 3 for this course.